packages/block-library/src/spacer/edit.js

```diff
@@ -6,24 +6,28 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { InspectorControls } from '@wordpress/block-editor';
-import { BaseControl, PanelBody, ResizableBox } from '@wordpress/components';
+import { PanelBody, ResizableBox, RangeControl } from '@wordpress/components';
 import { compose, withInstanceId } from '@wordpress/compose';
 import { withDispatch } from '@wordpress/data';
 
+const minSpacerHeight = 20;
+const maxSpacerHeight = 500;
+
 const SpacerEdit = ( {
 	attributes,
 	isSelected,
 	setAttributes,
-	instanceId,
 	onResizeStart,
 	onResizeStop,
 } ) => {
 	const { height } = attributes;
-	const id = `block-spacer-height-input-${ instanceId }`;
-	const [ inputHeightValue, setInputHeightValue ] = useState( height );
+	const changeAttribute = ( value ) => {
+		setAttributes( {
+			height: value,
+		} );
+	};
```

Comment:
Using a standard function declaration has the same effect while being clearer (it's more obviously a function), and being a few characters shorter.
```suggestion
	function changeAttribute( value ) {
		setAttributes( {
			height: value,
		} );
	}
```

Reply:
@ZebulanStanphill I would tend to disagree here.
Using `function` or assigning an anonymous function to a `const` is not the same, the main difference being the time at which they are defined and the fact that a `const` cannot be changed. The `function` keyword basically works like `var` which is discouraged in the [WordPress Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#declaring-variables-with-const-and-let) for ESNext. That said there is nothing really explicit about it in the docs so we should use them at our convenance, remembering the differences between the two